src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Current Release
 
+## Version 0.9.11
```

Comment:
@yangyuan please remove this version header you added -- you can place the below line under the `## Current Release` header

Reply:
Done